src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/util/XMLStringUtil.java

```diff
@@ -0,0 +1,32 @@
+package ca.bc.gov.open.ecrc.util;
```

Comment:
Would it make sense to move this Util class into src/test/java since its used only for testing?

Reply:
I moved it. I can't see a time when we would use it otherwise.